src/github/client/github-queries.ts

```diff
@@ -112,12 +112,12 @@ export type getCommitsResponse = {
 	}
 };
 
-export const getCommitsQueryWithChangedFiles = `query ($owner: String!, $repo: String!, $per_page: Int!, $cursor: String) {
+export const getCommitsQueryWithChangedFiles = `query ($owner: String!, $repo: String!, $per_page: Int!, $commitSince: GitTimestamp, $cursor: String) {
   repository(owner: $owner, name: $repo){
     defaultBranchRef {
       target {
         ... on Commit {
-          history(first: $per_page, after: $cursor) {
+          history(first: $per_page, after: $cursor, since: $commitSince) {
```

Comment:
feature flag (here and everywhere in this file)?

Reply:
Soo... I see it is passing as undefined in case `timeCutoffMsecs` is not provided. Wondering how this query would work in this case?

I think we should include this variable only if it is provided, or is graphql protocol will understand `undefined`?

Reply:
I guess if you do this, no FF is needed (because we are essentially switching it in the caller and when off the old code will work).

Reply:
Well, we change it. Even though it is undefined, we send the modified query + a new key in the variables, relying/hoping GitHub will work as we expect.  FF should allow us to rollback to the old behaviour fast. With this change this is impossible.

Reply:
...we can disable the FF and get the previous behavior though.  This has been confirmed through unit tests and pollinator.  Just because any code has changed doesn't mean _everything_ needs to be behind a FF.  If that was the case, half the codebase would be duplicated.  It doesn't make logical sense to do this and will add more code/complexity to try to FF everything behind it

Reply:
Discussed offline, summary:
1. this FF will stay for a while, therefore duplicating code is bad (not a temp code)
2. we will be enabling FF not for everyone but just for canaries at start, therefore this change will be tested on them anyway

Reply:
Actually no, I'm wrong, the query will be changed for everyone. Ok, let's see how it go anyway 🍿